pkg/dao/suite_test.go

```diff
@@ -122,10 +123,10 @@ func (s *RpmSuite) SetupTest() {
 	})
 	s.tx = s.db.Begin()
 
-	// Remove the content for the 3 involved tables
-	s.tx.Where("1=1").Delete(models.Rpm{})
-	s.tx.Where("1=1").Delete(models.Repository{})
-	s.tx.Where("1=1").Delete(models.RepositoryConfiguration{})
+	err := models.DropAll(s.tx)
+	if err != nil {
+		s.FailNow(err.Error())
+	}
```

Comment:
I moved all of these delete calls to a single method for a couple reasons:
1) repeated code
2) dropping repository before repo config was actually a mistake and would error
3) errors werent' being caught and so 2) caused really weird behavior as the transaction would be aborted yet the code would still run